setup.py

```diff
@@ -76,12 +74,12 @@
     include_package_data=True,
     install_requires=[
         "eth-ape>=0.8.12,<0.9",
```

Comment:
this will fix CI/CD I think

```suggestion
        "eth-ape>=0.8.21,<0.9",
```